docs/src/main/docs/guides/07_datasource.adoc

```diff
@@ -0,0 +1,364 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Setting Up Data Sources
```

Comment:
This document isn't really a guide per the current definition. (create a project, do something, exercise something).
Maybe this can moved to the normal document as-is.

Reply:
Hmm.  OK.  This document is a prerequisite for the JPA guide too.  So that's now on hold.

How would you suggest I improve this document's structure?

Reply:
How about splitting this into 2 documents, one for each data source and document the steps needed to exercise each data source in a small project.
Not sure how that fits with the JPA guides, maybe these are multi-part guides ? (these would be part1, the jpa guide would be part 2?)